example_test.go

```diff
@@ -913,12 +917,51 @@ func ExampleFuture_GetIterator() {
 }
 
 func ExampleConnect() {
-	conn, err := tarantool.Connect("127.0.0.1:3013", tarantool.Opts{
-		Timeout:     5 * time.Second,
-		User:        "test",
-		Pass:        "test",
-		Concurrency: 32,
-	})
+	ctx, cancel := context.WithTimeout(context.Background(), 500*time.Millisecond)
+	defer cancel()
+
+	conn, err := tarantool.Connect(ctx, "127.0.0.1:3013",
+		tarantool.Opts{
+			Timeout:     5 * time.Second,
+			User:        "test",
+			Pass:        "test",
+			Concurrency: 32,
+		})
```

Comment:
The same about the codestyle.
```suggestion
	conn, err := tarantool.Connect(ctx, "127.0.0.1:3013", tarantool.Opts{
		Timeout:     5 * time.Second,
		User:        "test",
		Pass:        "test",
		Concurrency: 32,
	})
```

Reply:
Updated.